src/snapshotter.js

```diff
@@ -94,12 +101,25 @@ export default class SnapShotter {
     }
   }
 
+  async executeScript(script) {
+    try {
+      const scriptToExecute = require(path.resolve(script));
+      await scriptToExecute(this._driver);
+    } catch (error) {
+      logger.error(
+        'snapshotter',
+        `❌  Unable to find the specified script location! ❌ ${error}`
```

Comment:
can we change this to be something like 

`unable to run script due to: `

I think there might be cases were we cant execute due to things other than locations

Reply:
updated